### ISSUE TITLE
Inconsistent parameters in the event object

### ISSUE THREAD

**Issue by jpswade** [serverless/serverless#5637]
# This is a Bug Report

## Description

The documentation here:

* https://github.com/serverless/serverless/blob/a891a9dca51768b7711cb1a35415cb78ada86d33/docs/guide/overview-of-event-sources.md#http-endpoint

Describes a "query" parameter, which doesn't appear to exist.

Yet the documentation here:

* https://serverless.com/framework/docs/providers/aws/events/apigateway#default-request-templates

Describes a different set of properties on the event object.

Which is correct? When did it change? Why did it change?

**Comment by dschep** [comment]
Yup, those don't seem quite accurate. See the sample event from AWS for reference: https://docs.aws.amazon.com/lambda/latest/dg/eventsources.html#eventsources-api-gateway-request

**Comment by dschep** [comment]
@pmuens what does this mean?
> Note: Serverless supports a built in, universal velocity request template which makes the following parameters available in the event object:
